records_mover/db/mysql/load_options.py

```diff
@@ -17,29 +16,17 @@
     MySqlCharacterSet = Literal['big5', 'binary', 'latin1', 'ucs2',
                                 'utf8', 'utf8mb4', 'utf16', 'utf16le',
                                 'utf32']
-
-    # The NoReturn annotations are the mypy way of validating we're
-    # covering all officially approved cases of an the hint at
-    # typechecking time.
-    #
-    # https://github.com/python/mypy/issues/6366#issuecomment-560369716
-    def invalid_hint(fail_if_cant_handle_hint: bool,
-                     hint_name: str,
-                     hints: RecordsHints,
-                     the_bad_hint: NoReturn) -> NoReturn:
-        assert False
-
 else:
     MySqlCharacterSet = str
 
-    # The cant_handle_hint at runtime is to cover cases that slip
-    # through due to our use of "type: ignore" rather than run-time
-    # validation below.
-    def invalid_hint(fail_if_cant_handle_hint: bool,
-                     hint_name: str,
-                     hints: RecordsHints,
-                     the_bad_hint: NoReturn) -> NoReturn:
-        cant_handle_hint(fail_if_cant_handle_hint, hint_name, hints)
+
+# The NoReturn annotations are the mypy way of validating we're
+# covering all officially approved cases of an the hint at
+# typechecking time.
+#
+# https://github.com/python/mypy/issues/6366#issuecomment-560369716
+def _assert_never(the_bad_hint: NoReturn) -> NoReturn:
+    assert False
```

Comment:
Since these hints are prevalidated, we should simply never hit the 'else' cases here rather than them representing an unknown hint value.